github-action/README.md

```diff
@@ -2,238 +2,4 @@
-        output: generated-html
-      
-    #Using another action that takes generated HTML and pushes it to GH Pages
-    - name: Deploy GH page
-      uses: JamesIves/github-pages-deploy-action@3.4.2
-      with:
-        ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        BRANCH: gh-pages
-        FOLDER: generated-html
-```
-
-### Example workflow for generating models
-
-In case you want to use models generated from your AsyncAPI document, you can use this action to generate them and then use them in your workflow. This is how full workflow could look like:
-
-```yaml
-
-name: AsyncAPI documents processing
-
-on:
-  push:
-    branches: [ master ]
-
-jobs:
-  generate-models:
-    runs-on: ubuntu-latest
-    steps:
-    #"standard step" where repo needs to be checked-out first
-    - name: Checkout repo
-      uses: actions/checkout@v2
-      
-    - name: Generating models from my AsyncAPI document
-      uses: asyncapi/github-action-for-cli@v3.1.1 # You can use any version you want
-      with:
-        command: generate
-        filepath: docs/api/my-asyncapi.yml
-        language: typescript
-        output: generated-models
-```
-
-### Example workflow for validating AsyncAPI document changes
-
-In case you want to validate your AsyncAPI document changes, you can use this action to validate them and then use them in your workflow. This is how full workflow could look like:
-
-```yaml
-name: Validate AsyncAPI document
-
-on:
-  pull_request:
-    branches: [ master ]
-
-jobs:
-  validate:
-    runs-on: ubuntu-latest
-    steps:
-    #"standard step" where repo needs to be checked-out first
-    - name: Checkout repo
-      uses: actions/checkout@v2
-      
-    - name: Validating AsyncAPI document
-      uses: asyncapi/github-action-for-cli@v3.1.1 # You can use any version you want
-      with:
-        command: validate
-        filepath: docs/api/my-asyncapi.yml
-```
-
-## Local dry run
-
-Use following commands to run and test github action locally:
-
-1. Build docker image of github action for cli
-
-  ```bash
-    npm run action:docker:build
-  ```
-
-2. Execute docker image with proper arguments
-
-  ```bash
-    docker run -e GITHUB_WORKSPACE="" --workdir /action  -v "/home/{user}/path/to/repo":"/action" asyncapi/github-action-for-cli  "" "generate" "github-action/test/asyncapi.yml" "@asyncapi/markdown-template@0.10.0" "" "output" "" ""
-  ```
-
-Make sure to change the path of the repo and user in the command.
-
-## Troubleshooting
-
-You can enable more log information in GitHub Action by adding `ACTIONS_STEP_DEBUG` secret to repository where you want to use this action. Set the value of this secret to `true` and you''ll notice more debug logs from this action.
-
-
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+The detailed docs can be found [here](../docs/github-action.md).
```

Comment:
better link to docs on website

link will be https://www.asyncapi.com/docs/tools/cli/installation/github-action